src/components/Imprint/ImprintContent.tsx

```diff
@@ -0,0 +1,54 @@
+import { DefaultTypography as Typography, DefaultLink as Link } from '@common/components'
+import { useTranslation } from 'react-i18next'
+export const ImprintContent = () => {
+  const { t } = useTranslation()
+  return (
+    <div>
+      <Typography variant="h4">{t('components.ImprintContent.Title')}</Typography>
+      <p></p>
+      <Typography variant="h6">{t('components.ImprintContent.Subtitle')}</Typography>
+      <Typography variant="body2">{t('components.ImprintContent.AsOf')} 26.04.2023</Typography>
+      <p></p>
```

Comment:
Wenns nicht anders geht (also MUI component schaut nicht aus wie <p/>) dann gehts auch so:

```suggestion
      <p/>
```